src/components/table.js

```diff
@@ -4,7 +4,7 @@ import React, {useState, useEffect} from 'react';
 import {Link} from 'react-router-dom';
 
 const isEqual = () => {
-  return true;
+  return false;
```

Comment:
Might be better to compare the props to check if they're unchanged instead of always returning false. Otherwise, it would lead to extra renders.